### DISCUSSION TITLE
Discord Group for Serverpod

### DISCUSSION THREAD

**Discussion by Vedsaga** [serverpod/serverpod#169] [General]
It would be nice to have Severpod  group on Discord Server :). I guess that we'll help new dev ask for quick to many developers. Because, as a developer many people hardly open Github frequently in a day. I hope you would consider this idea. Consider me new to pod family.

**Comment by MarkOSullivan94** [comment]
The only problem with a Discord server is that useful information can be lost in the chat which would be far more easily found on GitHub's Issue or Discussions page

**Reply by Vedsaga** [reply]
I agree, but I feel because communicating will be mainly more active on Discord on then Github discussion so, that would be advantages. As mainly most people don't open Github every now and then

**Comment by vlidholt** [comment]
I think we may want a Discord or Slack at some point, but we may hold off until the community is a bit larger.

**Reply by ordru** [reply]
I agree with the concerns. However, on the other side, having Discord (or Slack, etc)  should be part of the strategy to a) **engage** and b) **grow** the community. It provides another venue of community engagement and may actually help retain more developers, even in the early stages like now. As practical example, it may help the active developers exchange information (e.g clarifications) related to ongoing development (e.g changes) that may not get into an issue or discussion or comment otherwise. 

For example, I like the project, I would like to the best of my abilities and time available, to participate and contribute to the project. However, at this time, I find that getting up to speed and engage with other participants, consumes more time than it should and it's here where direct communication would be useful. Yes, I understand the overhead that direct communication sometime entails, however, setting expectations and guiding the community to use effective communication, would reduce overhead. Just some thoughts from my side :)
